README.md

```diff
@@ -122,15 +122,9 @@ Run "npx stylelint "**/*.{css,scss}" --fix " to fix linters
 
 ### Deployment
 
-You can deploy this project using:
+You can deploy this project using:https://god-logic.github.io/My-Portfolio-Website/
```

Comment:
[OPTIONAL]
- [ ] Nice work deploying your application everything works fine :+1: However It will be better to add the link to the "Live Demo" section
E.g:
## 🚀 Live Demo <a name="live-demo"></a>

> Add a link to your deployed project.

- [Live Demo Link](https://god-logic.github.io/My-Portfolio-Website/)